CMakeLists.txt

```diff
@@ -104,6 +104,32 @@ include_directories(
     ${Qt5Widgets_INCLUDES}
 )
 
+# Mac OS X Bundle infomations
+if (APPLE)
```

Comment:
I was thinking about moving this entire `if` block to its own file, located at `/cmake`, since it's a bit long and apple only.

(If that doing that is too much hassle, then it's also OK to keep it here.)